code/datums/mind.dm

```diff
@@ -1085,22 +1101,24 @@
 				if(has_antag_datum(/datum/antagonist/traitor))
 					to_chat(current, "<span class='warning'><FONT size = 3><B>You have been brainwashed! You are no longer a traitor!</B></FONT></span>")
 					remove_antag_datum(/datum/antagonist/traitor)
-					current.client.chatOutput?.clear_syndicate_codes()
```

Comment:
This got removed

Reply:
It has been moved to line 58 of `datum_traitor.dm`.